lansa-vmss-windows-autoscale-sql-database/azuredeploy.json

```diff
@@ -0,0 +1,829 @@
+                  "name": "[variables('dbbePoolName')]"
+               }
+            ],
+            "loadBalancingRules": [
+               {
+                  "name": "lbrule",
+                  "properties": {
+                     "frontendIPConfiguration": {
+                        "id": "[concat(variables('dblbID'),'/frontendIPConfigurations/loadBalancerFrontEnd')]"
+                     },
+                     "backendAddressPool": {
+                        "id": "[concat(variables('dblbID'), '/backendAddressPools/', variables('dbbePoolName'))]"
+                     },
+                     "probe": {
+                        "id": "[concat(variables('dblbID'), '/probes/lbprobe')]"
+                     },
+                     "protocol": "Tcp",
+                     "loadDistribution": "SourceIP",
+                     "frontendPort": 50000,
+                     "backendPort": 3389,
+                     "idleTimeoutInMinutes": 15
+                  }
+               }
+            ],
+            "probes": [
+               {
+                  "name": "lbprobe",
+                  "properties": {
+                     "protocol": "tcp",
+                     "port": 3389,
+                     "intervalInSeconds": 15,
+                     "timeoutInSeconds": 31,
+                     "numberOfProbes": 2
+                  }
+               }
+            ]
+         }
+      },
+      {
+         "type": "Microsoft.Compute/virtualMachineScaleSets",
+         "name": "[variables('dbvmssName')]",
+         "location": "[resourceGroup().location]",
+         "apiVersion": "2016-03-30",
+         "dependsOn": [
+            "storageLoop",
+            "[concat('Microsoft.Storage/storageAccounts/', variables('diagnosticsStorageAccountName'))]",
+            "[concat('Microsoft.Network/loadBalancers/', variables('loadBalancerName'))]",
+            "[concat('Microsoft.Network/virtualNetworks/', variables('virtualNetworkName'))]",
+            "[concat('Microsoft.Sql/servers/', variables('sqlserverName'))]",
+            "[concat('Microsoft.Sql/servers/', variables('sqlserverName'), '/databases/', parameters('databaseName'))]"
+         ],
+         "sku": {
+            "name": "[variables('dbvmSku')]",
+            "tier": "Standard",
+            "capacity": "1"
+         },
+         "plan": {
+            "name": "[variables('imageSKU')]",
+            "product": "[parameters('imageOffer')]",
+            "publisher": "[variables('imagePublisher')]"
+         },
+         "properties": {
+            "overprovision": "false",
+            "upgradePolicy": {
+               "mode": "Automatic"
+            },
+            "virtualMachineProfile": {
+               "storageProfile": {
+                  "osDisk": {
+                    "vhdContainers": [
+                      "[concat('https://', variables('uniqueStringArray')[0], variables('newStorageAccountSuffix'), '.blob.core.windows.net/', variables('vhdContainerName'))]",
```

Comment:
please see the best practice around "all cloud" uris: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/bp-checklist.md